### PR TITLE
Fixes #13797 - Update candlepin consumer on host update

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: app/presenters/katello/product_content_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Katello
+  class ProductContentPresenter
```
Comment (ehelms) [inline app/presenters/katello/product_content_presenter.rb]: Mis-commit? Doesn't appear to be used in this PR.

File: app/lib/actions/katello/host/update.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@ def plan(host, consumer_params = nil)
               plan_action(::Actions::Candlepin::Consumer::Update, host.subscription_facet.uuid, consumer_params)
             elsif host.subscription_facet
               consumer_params = host.subscription_facet.consumer_attributes
+              plan_action(::Actions::Candlepin::Consumer::Update, host.subscription_facet.uuid, consumer_params)
```
Comment (ehelms) [inline app/lib/actions/katello/host/update.rb]: Maybe another layer of conditional logic to DRY up this plan_action based upon whether `host.subscription_facet` is true?

File: app/presenters/katello/product_content_presenter.rb (hunk 1 above)
Comment (jlsherrill) [inline app/presenters/katello/product_content_presenter.rb]: oops, yes!

**Comment by jlsherrill** [comment]
DRY'd it up a little and added correct file for test versus that other file :)

**Comment by ehelms** [comment]
ACK

**Pull request merged**